endpoints/openrtb2/openrtb2_go111_test.go

```diff
@@ -0,0 +1,23 @@
+// +build go1.11
```

Comment:
Can you split this Go 1.11 fix off into a separate PR?  It's a valuable change... but commits do get squashed on merge, and it's unrelated.

Small note: I'd prefer something like `message_go1.11` rather than `message_alt`. We already know that this bug has been fixed in Go 1.11.1 (see https://github.com/golang/go/issues/27275), so... it's a pretty specific fix.

Would also be nice to lin this issue somewhere in the file so we remember why the workaround exists.

Reply:
I was thinking the same thing for the message name. I went with message_alt because it will apply to go1.11+, now that I know it was an actual bug and it's been fix it makes a lot more sense to do message_go1.11.